app/models.py

```diff
@@ -79,10 +97,13 @@ class Masterclass(db.Model):
     room = db.Column(db.String, index=True)
     floor = db.Column(db.String, index=True)
     building_instructions = db.Column(db.String, index=True)
-    masterclass_content_id = db.Column(db.Integer, db.ForeignKey('masterclass_content.id'))
-    location_id = db.Column(db.Integer, db.ForeignKey('location.id'))
-    instructor_id = db.Column(db.Integer, db.ForeignKey('user.id'))
-    attendees = db.relationship('MasterclassAttendee', backref='masterclass')
+    masterclass_content_id = db.Column(
```

Comment:
Can we remove the foreign key now we've got the relationship?

Reply:
Nope - and this is why there's no migration file. The relationship is established and manipulated at the Python level, rather than at the database. Having a relationship field is just a bit of magic syntax to link Python objects together, while the FK is a database-level link of objects.

Reply:
Ahhh cool i didn't know that! Thanks :D